.gitignore

```diff
@@ -4,3 +4,4 @@ coverage.txt
 profile.out
 containerd.test
 _site/
+.idea/
```

Comment:
please remove this and place it in your global gitignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
@crosbymichael ok , thank you !